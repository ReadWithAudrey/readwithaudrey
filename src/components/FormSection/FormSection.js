import React from 'react'
import PropTypes from 'prop-types'
import { StyledForm } from './FormSection.style'

const FormSection = props => {
  return <StyledForm>{props.children}</StyledForm>
}

FormSection.propTypes = {
  children: PropTypes.array,
}

export default FormSection
