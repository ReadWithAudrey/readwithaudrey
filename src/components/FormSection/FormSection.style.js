import styled from 'styled-components'

export const StyledForm = styled.section.attrs({
  className: 'b--green',
})`
  @media (min-width:801px)  { border: 3px solid; }
color: #000;
  }
`
