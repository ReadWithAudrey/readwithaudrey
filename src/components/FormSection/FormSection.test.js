import React from 'react'
import renderer from 'react-test-renderer'
import FormSection from './FormSection'

describe('FormSection component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<FormSection />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
