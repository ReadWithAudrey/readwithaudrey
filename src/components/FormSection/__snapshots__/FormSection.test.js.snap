// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormSection component snapshot test renders correctly 1`] = `
<section
  className="sc-bdVaJa jxafWn"
/>
`;
